calamari_ocr/utils/image.py

```diff
@@ -95,3 +93,26 @@ def to_uint8(data: np.ndarray) -> np.ndarray:
         raise Exception("unknown image type: {}".format(data.dtype))
 
     return data
+
+
+def to_float32(data: np.ndarray) -> np.ndarray:
+    """Read an image and returns it as float32
+
+    All arrays are rescaled to the range 0...1
+    """
+    if data.dtype == np.dtype("uint8"):
+        data = data.astype("float32") / 255
+    elif data.dtype == np.dtype("int8"):
+        data = (data.astype("int16") + 128).astype("float32") / 255
+    elif data.dtype == np.dtype("uint16"):
+        data = data.astype("float32") / 256 / 255
```

Comment:
I'm a bit confused by the 256 here: max of uint16 should be 65535, divided by 255 equals 257. It's 35 degrees in here so I might be wrong...

Reply:
You are totally right, I wasn't thinking. Fixed it!

Reply:
Also for int16 (please verify, 35 seems to be good for your brain :-) )

Reply:
Looks good now!